web/customize_viewer.css

```diff
@@ -55,3 +184,28 @@
     margin-left: 72px;
   }
 }
+
+/**
+ * Implement our custom icons. Disable all of the default icons, since the
+ * buttons now have inline svg in viewer.html.
+ */
+.toolbarButton::before {
+  content: '' !important;
```

Comment:
It is enough to hide it, but the `content: ''` is necessary to prevent network requests from rules like `content: url(images/toolbarButton-openFile.png);`